mm/flow.py

```diff
@@ -50,11 +53,26 @@ def alignment_flow(feature_net, alignment_cache_path=None):
     aggregate = net.add_node("generic-aggregation-vector-f32", "aggregate")
     net.link(mapping[feature_net.get_output_links("features").pop()], aggregate)
 
-    alignment = net.add_node(
-        "speech-alignment",
-        "alignment",
-        {"id": "$(id)", "orthography": "$(orthography)"},
-    )
+    if add_left_right_context_orth:
```

Comment:
similarly to the factoring out above one could always add the base form of the alignment node and then if add_left_right_context_orth update the node.. but I don't know from the top of my head how to do it.

Probably something like `net.nodes[alignment].update({"left-context-orthography": "$(left-context-orthography)","right-context-orthography": "$(right-context-orthography)"})`